yamllint/config.py

```diff
@@ -96,6 +102,12 @@ def parse(self, raw_content):
             self.ignore = pathspec.PathSpec.from_lines(
                 'gitwildmatch', conf['ignore'].splitlines())
 
+        if 'yaml-files' in conf:
+            if not isinstance(conf['yaml-files'], list):
+                raise YamlLintConfigError(
+                    'invalid config: yaml-files should be a list of file patterns.')
```

Comment:
Other yamllint errors don't have `.` at the end, could you remove this one to be consistent?

Reply:
adjusted